less/bottom-content.less

```diff
@@ -0,0 +1,267 @@
+.home .bottom-content {  
+    padding-bottom: 10%;
+    padding-top: 10%;
+
+    @media @phone {
+      margin: 0 -8%;
+    }
+  
+    h1 {
+      .sub-header();
+      
+      
+
+      @media @phone {
+        padding-left: 7%;
+      }
+    }
+    
+    .location-box {
+      display: flex;
+      flex-direction: column;
+      justify-content: flex-start; 
+      padding-top: 10%; 
+          
+  
+  
+      .overlay-1 {
+        display: flex;
+        justify-content: flex-end;
+        align-items: center;
+        height: 34vh;      
```

Comment:
Ah, here's the issue. You're setting the size of your image to vh. Remember, that will change based on the size of the viewport! Not what you want. So while it might look good on your screen like this, on a smaller or larger screen it will completely change the proportions.